lib/phoenix/live_dashboard/router.ex

```diff
@@ -22,6 +22,17 @@ defmodule Phoenix.LiveDashboard.Router do
     * `:live_socket_path` - Configures the socket path. it must match
       the `socket "/live", Phoenix.LiveView.Socket` in your endpoint.
 
+    * `:historical_data` - Configures a callback for retreiving metric history.
+      It must be a map with lists of atoms as keys and
+      tuples of {Module, :function, list} as values such as
+
+      historical_data: %{
+        [:namespace, :metric] =>
+          {MyStorage, :historical_metric_data, []}
+      }
+      If not set, metrics will start out empty/blank and only display
+      data that occurs while the browser page is open.
```

Comment:
These docs are out of date. Also, please use four spaces to indent the code.

Reply:
This is not showing as outdated, probably because the line changed was 30-32 not 34, but is resolved